README.adoc

```diff
@@ -59,8 +59,8 @@ environments. It handles scheduling, deployment, as well as mass deletion and cr
 It provides update rollout capabilities on a large scale that would otherwise prove extremely tedious
 to do. Imagine that you updated a Docker image, which now needs to propagate to a dozen containers.
 While you could destroy and then recreate these containers, you could also issue a short one-line
-command and have {kube} do all that updating for you. Of course this is just a simple example, the
-iceberg that is {kube} has a lot more to offer.
+command and have {kube} do all that updating for you. Of course this is just a simple example. The bigger value add,
```

Comment:
Maybe it would be better to simplify this to:
> Of course this is just a simple example. {kube} has a lot more to offer.

Does that work?

Reply:
I like "Of course this is just a simple example. {kube} has a lot more to offer."